cmd/fleetctl/get.go

```diff
@@ -451,6 +450,7 @@ func getQueriesCommand() *cli.Command {
 							MinOsqueryVersion:  query.MinOsqueryVersion,
 							AutomationsEnabled: query.AutomationsEnabled,
 							Logging:            query.Logging,
+							DiscardData:        query.DiscardData,
```

Comment:
good call, fleetctl updates feel easy to miss.  Maybe we can add it to https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Adding-new-endpoints.md

Reply:
Actually these were captured by failing tests IIRC. But yes, worth documenting.